### PR TITLE
Rename Particle.bar to Particle.is_name_barred

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Another small change to the `Particle` API, @henryiii. Seems better.

All for the imminent release 0.4.0.

### CODE REVIEW

**Comment by henryiii** [comment]
How about is_barred or is_bar? This seems long and a bit too wordy. I do think the "is" is useful, so a user doesn't try to add a bar with it.

**Comment by eduardo-rodrigues** [comment]
It's a bit wordy but having "name" in the method name does say that we're talking about particle names. It's a method code can handily use but I do not see users needing it very often. I would still prefer is_name_barred,so replacing has with is :-).
